lib/adaptors/serialport.js

```diff
@@ -52,13 +52,13 @@ util.inherits(Adaptor, EventEmitter);
  */
 Adaptor.prototype.open = function open(callback) {
   var self = this,
-      port = this.serialport = new serialport.SerialPort(this.conn, {});
+      port = this.serialport = new serialport.SerialPort(this.conn, {}, false);
 
   function emit(name) {
     return self.emit.bind(self, name);
   }
 
-  port.on("open", function(error) {
+  port.open(function(error) {
```

Comment:
This change makes the first test fail.
@jamuus seems like SerialPort doesn't support open()

```
  1) Serialport Adaptor #open "before each" hook:
     TypeError: port.open is not a function
      at Adaptor.open (lib/adaptors/serialport.js:61:8)
      at Context.<anonymous> (spec/lib/adaptors/serialport.spec.js:41:15)
```